src/constants/responses.ts

```diff
@@ -13,3 +15,12 @@ export const BAD_SIGNATURE = {
 export const STATUS_CHECK = {
   message: "Welcome to our discord Bot Server 👋",
 };
+
+export const DISCORD_RESPONSE = (reply: string) => {
```

Comment:
Can a function name be written as constants ??

Reply:
This should not be part of the `Constant` file

Reply:
ok, will have a look into it